packages/gatsby/src/utils/gatsby-cloud-config.ts

```diff
@@ -0,0 +1,18 @@
+import { IGatsbyConfig } from "../internal"
+
+type ConstructConfigObjectResponse = Pick<
+  IGatsbyConfig,
+  "trailingSlash" | "assetPrefix" | "pathPrefix"
+>
+
+export function conststructConfigObject(
+  gatsbyConfig: IGatsbyConfig
+): ConstructConfigObjectResponse {
+  return {
+    trailingSlash: gatsbyConfig.trailingSlash ?? `legacy`,
```

Comment:
Does `trailingSlash` have a `legacy` value? How does it compare with `ignore`?

Reply:
Yes, it has a `legacy` value. On Cloud it should behave the same as `ignore` for now, we'll remove `legacy` in Gatsby v5